rules/private/phases/phase_javainfo.bzl

```diff
@@ -49,11 +49,16 @@ def phase_javainfo(ctx, g):
                 java_toolchain = find_java_toolchain(ctx, ctx.attr._java_toolchain),
             )
 
+        source_jar_name = ctx.outputs.jar.basename.replace(".jar", "-src.jar")
```

Comment:
I don't remember right now if we use `-src.jar` or -sources.jar`. I think I'm good either way, but we might as well keep doing whatever we're doing now.

Reply:
IIUC, -src.jar is what it was making before.